include/swift/Basic/Features.def

```diff
@@ -44,5 +44,6 @@ LANGUAGE_FEATURE(GlobalActors, 0, "Global actors", langOpts.EnableExperimentalCo
 LANGUAGE_FEATURE(BuiltinJob, 0, "Builtin.Job type", true)
 LANGUAGE_FEATURE(Sendable, 0, "Sendable and @Sendable", true)
 LANGUAGE_FEATURE(BuiltinContinuation, 0, "Continuation builtins", true)
+LANGUAGE_FEATURE(BuiltinTaskGroup, 0, "TaskGroup builtins", true)
```

Comment:
ah, nice... Shall I do the same for `spawn let`?

Reply:
It's only necessary if you're planning to use the builtins in inlinable Swift code somewhere.  Otherwise they don't show up in the `.swiftinterface` and shouldn't bother old compilers.

Reply:
I see, thanks for clarifying 👍